speechbrain/lobes/models/transformer/Conformer.py

```diff
@@ -17,11 +20,48 @@
     MultiheadAttention,
     PositionalwiseFeedForward,
 )
+from speechbrain.lobes.models.transformer.DCT import DCTConfig
 from speechbrain.lobes.models.transformer.hypermixing import HyperMixing
 from speechbrain.nnet.normalization import LayerNorm
 from speechbrain.nnet.activations import Swish
 
 
+@dataclass
+class ConformerEncoderLayerStreamingContext:
+    """Streaming metadata and state for a `ConformerEncoderLayer`.
+
+    The multi-head attention and Dynamic Chunk Convolution require to save some
```

Comment:
Should we add the Dynamic Chunk Convoluation paper as a reference here .

Reply:
Made it refer to `ConvolutionModule` which now cites the paper.